src/main/java/com/genfood/foodgenback/endpoint/rest/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.genfood.foodgenback.endpoint.rest.controller;
+
+import com.genfood.foodgenback.endpoint.rest.controller.validator.MailValidator;
+import com.genfood.foodgenback.model.User;
+import com.genfood.foodgenback.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+public class UserController {
+  private final UserService userService;
+  private final MailValidator mailValidator;
+
+  @PutMapping("/user")
+  public User Save(@RequestBody User userEntity) {
```

Comment:
it should take a List of User to create or update and return a list of User created or updated.

Save -> crupdateUsers
User -> List<User>
@RequestBody User userEntity -> @RequestBody List<User> users